.vale/styles/RedHat/PascalCamelCase.yml

```diff
@@ -0,0 +1,51 @@
+---
+extends: existence
+scope: sentence
+ignorecase: false
+nonword: true
+level: warning
+link: https://redhat-documentation.github.io/asciidoc-markup-conventions
+message: "Consider wrapping this Pascal case or Camel case term ('%s') in backticks."
+source: https://github.com/redhat-documentation/vale-at-red-hat/tree/main/.vale/styles/RedHat/PascalCamelCase.yml
+tokens:
+  - ([A-Z]+[a-zA-Z]+){2,}
+  - ([a-zA-Z]+([A-Z][a-z]+))+
+  - Alertmanager
+  - Authentication
+  - Binding
+  - Build
+  - Config
+  - Console
+  - Deployment
+  - Endpoints
+  - Etcd
+  - Event
+  - Group
+  - Help
+  - Identity
+  - Image
+  - Infrastructure
+  - Ingress
+  - Lease
+  - Namespace
+  - Network
+  - Node
+  - OAuth
+  - Pod
+  - Probe
+  - Profile
+  - Project
+  - Prometheus
+  - Provisioning
+  - Proxy
+  - Role
+  - Route
+  - Scheduler
+  - Secret
+  - Storage
+  - Subscription
```

Comment:
This would trigger a lot of false positives for Subscription Manager in RHEL documentation (78).